src/combinatorics/composition.lean

```diff
@@ -344,6 +344,7 @@ lemma disjoint_range {i₁ i₂ : fin c.length} (h : i₁ ≠ i₂) :
 begin
   classical,
   wlog h' : i₁ ≤ i₂ using i₁ i₂,
+  swap, exact (this h.symm).symm,
```

Comment:
It's not important for merging this PR, but do you know why the behavior of this proof changed? Is it something we can fix in the `wlog` tactic?

Reply:
I believe this is due to https://github.com/leanprover-community/lean/pull/579

Reply:
We may as well do this while we're at it.
```suggestion
  swap, { exact (this h.symm).symm },
```
LGTM otherwise, is there anything else this is waiting for?

Reply:
I also thought about the braces but they seem a bit superfluous with exact.

Otherwise, this PR is ready.